content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -109,13 +110,101 @@ Each HTTP rule contains the following information:
 A default backend is often configured in an Ingress controller to service any requests that do not
 match a path in the spec.
 
-### Default Backend
+### Path Types
+
+Each path in an Ingress has a corresponding path type. There are three supported
+path types:
+
+* **ImplementationSpecific:** With this default path type, matching is up to the
+  IngressClass. Implementations can treat this as a separate PathType or treat
```

Comment:
actually, also this nit:
```suggestion
  IngressClass. Implementations can treat this as a separate `pathType` or treat
```

(to match how it looks in a manifest)

Reply:
Updated, thanks!